scss/_utilities_has-icon.scss

```diff
@@ -0,0 +1,7 @@
+@mixin vf-u-has-icon {
+  // stylelint-disable-next-line selector-max-type
+  .u-has-icon > i:first-child {
```

Comment:
We shouldn't target `i` here, we don't know if that will be an icon.

I think original code proposed something with class name containing `p-icon--`? `[class*='p-icon--"]`

This would also allow to remove the stylelint exception